frontend/packages/ceph-storage-plugin/src/components/bucket-class/wizard-pages/review-page.tsx

```diff
@@ -54,7 +55,8 @@ const ReviewPage: React.FC<ReviewPageProps> = ({ state }) => {
           </ReviewListBody>
           <ReviewListBody hideIcon>
             <span>{t('ceph-storage-plugin~Time to live: ')}</span>&nbsp;
-            <span className="text-secondary">{`${timeToLive} ms`}</span>
+            {/* <span className="text-secondary">{`${timeToLiveInMs} ${timeUnit}`}</span> */}
+            <span className="text-secondary">{`${timeToLive} ${timeUnit}`}</span>
```

Comment:
nit - 
Pluralize this so that it shows 
1 hour
15 hours
15 mins 
Check how it is done in multi-namespace-store.tsx L84